tutorials/tutorial1_sinus-interactive.ipynb

```diff
@@ -4,10 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# DECOMON tutorial #1\n",
-    "\n",
+    " # <span style='background:purple'> <span style='color:White'> DECOMON tutorial #1 </span></span>\n",
+    " \n",
     "_**Bounding the output of a Neural Network trained on a sinusoidal function**_\n",
     "\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"data/decomon.png\" alt=\"Decomon!\" width=\"100\">\n",
```

Comment:
Beware the logo decomon.png is not exisiting (on the repo). There is only the "data/decomon.jpg".
Moreover it could be a good idea to only have one version of the logo on the repo. The doc has it here: docs/source/_static/banner.jpg, maybe we could point to it?